
Error:
Traceback (most recent call last):
  File "C:\AVRL\online-judge-v\online_judge\codes\fb951122-031f-404e-9988-4cb96c5ce522_code.py", line 13, in <module>
    num = int(input())
EOFError: EOF when reading a line
